scripts/detectNotUsingGitPush1by1.fsx

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env -S dotnet fsi
+
+open System
+open System.Net.Http
+open System.Net.Http.Headers
+
+#r "nuget: Fsdk, Version=0.6.0--date20230214-0422.git-1ea6f62"
+
+let currentBranch =
+    Fsdk
+        .Process
+        .Execute(
+            {
+                Command = "git"
+                Arguments = "rev-parse --abbrev-ref HEAD"
+            },
+            Fsdk.Process.Echo.All
+        )
+        .UnwrapDefault()
+        .Trim()
+
+let prCommits =
+    Fsdk
+        .Process
+        .Execute(
+            {
+                Command = "git"
+                Arguments =
+                    sprintf "rev-list %s~..%s" currentBranch currentBranch
+            },
+            Fsdk.Process.Echo.All
+        )
+        .UnwrapDefault()
+        .Trim()
+        .Split "\n"
+    |> Seq.tail
+
+let notUsingGitPush1by1 =
+    prCommits
+    |> Seq.map(fun commit ->
+        let client: HttpClient = new HttpClient()
```

Comment:
@realmarv is client IDisposable? if yes, then it should use `use`, not `let`; if not, then it shouldn't use `new` keyword (this comment applies to other line where you're also using `new`)

Reply:
Done